examples/api/lib/material/theme/theme_extension.1.dart

```diff
@@ -18,7 +18,7 @@ class MyColors extends ThemeExtension<MyColors> {
   final Color? red;
 
   @override
-  MyColors copyWith({Color? red, Color? blue}) {
+  MyColors copyWith({Color? blue, Color? red}) {
```

Comment:
would firstColor and secondColor be more generic for this example?

Reply:
this was meant to represent semantic colors, but you're right, I can change to something more meaningful like "caution" and "danger"